Exercise_B/index.html

```diff
@@ -13,17 +13,50 @@
       href="https://fonts.googleapis.com/css?family=Lato"
       rel="stylesheet"
     />
-    <!-- Custom Stylesheet -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x" crossorigin="anonymous">
     <link rel="stylesheet" href="./assets/style/stylesheet.css" />
   </head>
 
   <body>
     <!--Header (flower shop logo block element)-->
-    <!--Banner Jumbotron of the image-->
 
+    <nav class="navbar navbar-light bg-light">
+      <a class="navbar-brand" href="#">
+        <img src='./assets/images/logo.png'  alt="">
+      </a>
+    </nav>
+    <!--Banner Jumbotron of the image-->
+    <div
+  class=" P-10 text-center rounded mb-5 text-white "
+  style="background-image:url(./assets/images/flowershop.jpg); height: 720px;"
+>
+  <h1 class="mb-3 h2">Beautiful Flowers</h1>
+  <button type="button" class="btn btn-success">Learn More</button>
+  
+</div>
     <!--Four containers with img / text-->
     <!-- 4 containers -->
-
+    <Div class="containers" style="background-color: rgba(0, 255, 115, 0.74);height: auto;">
+      <Div class="row" style="text-align: center;">
+        <div class="col-sm">
+          <img src="./assets/images/flower-icon.png">
+          <p>Flower for every <br>kind of situation</p>
+        </div>
+          <div class="col-sm">
+            <img src="./assets/images/payment-icon.png">
+            <p>All major payment <br> kinds accepted</p>

```

Comment:
So in this exercise we are asking you to use the bootstrap column system to put all of the content on one line on an extra large screen
2 icons and their text per line on a medium screen
1 icon and its content per line on a small screen

to achieve this tell the elements what amount of rows they should be allocated using the bootstrap break points.

col-sm-12 col-md-6 col-xl-3
just add that too your elements.